src/app/complex/complex-results/complex-navigator/table-structure/table-interactor-column/table-interactor-column.component.ts

```diff
@@ -319,32 +319,45 @@ export class TableInteractorColumnComponent implements OnInit {
   }
 
   public displayTopLineClass(complex: EnrichedComplex, interactorIndex: number): string {
-    if (this.doesLineCrossInteractorCell(complex, interactorIndex) || this.doesLineEndOnInteractorCell(complex, interactorIndex)) {
+    if (this.doesLineCrossInteractorCell(complex, interactorIndex)) {
       return 'verticalLine';
     }
+    if (this.doesLineEndOnInteractorCell(complex, interactorIndex) && !this.doesLineStartOnInteractorCell(complex, interactorIndex)) {
```

Comment:
We only display the top part of the line if the line cross through this cell (if above), or if the line end in this cell, but it did not start here. It could start and end on the same cell if a complex has a single interactor.